shumai/util/memory.ts

```diff
@@ -0,0 +1,73 @@
+import { Tensor } from '../tensor/tensor'
+export const creationTracker: Map<number, Tensor> = new Map()
+export const altTracker: Record<number, Tensor> = {}
```

Comment:
remove I think?

Reply:
Ahh, good catch; remnant of some tinkering to try and maximize performance on bench runs.